### ISSUE TITLE
Anyone get wifi sniffing to work?

### ISSUE THREAD

**Issue by mark-hahn** [dnschneid/crouton#1738]
I've seen several posts here about people trying to capture wifi packets.  Has anyone actually succeeded at this with either internal or external wifi?  I've been trying to use wireshark with airmon-ng on a chromebook pixel 2015 and I'm only getting local traffic.

Does anyone know what hardware driver for wifi is on a pixel 2015?

**Comment by tedm** [comment]
@mark-hahn this link may help:

https://forums.kali.org/showthread.php?24248-Samsung-Chromebook-Kali-promiscuous-mode-wifi-wireless-airmon-ng

**Comment by tcagle** [comment]
wireshark works fine on my hp 14 (pavilion) chromebook. Never needed it
before but just tried it and it worked like a champ. No experience with
pixel though sorry.

**Comment by mark-hahn** [comment]
Thanks to both of you.  I guess my internal adapter isn't going to work.  I've tried everything.

Now to decide if it's worth it to get the external USB @tedm used.

@tedm:  I assume you used the hub because the chromebook usb didn't have enough power?

**Comment by tedm** [comment]
@mark-hahn I'm not using wireless right now, but in addition to the link above, there are some folks who  have been able to get their built in chromebook wi-fi adapters to go into promiscuous mode (HP I think), I will try to find that link.

My Samsung 2012 ARM is picky about chipsets for hubs and USB to Ethernet ports, but a couple out there work well in both chrome and the chroot, run dmesg and see if your device is being utilized:
[43787.894478] usb 2-1.1: new SuperSpeed USB device number 25 using xhci-hcd
[43787.910398] usb 2-1.1: Parent hub missing LPM exit latency info.  Power management will be impacted.
[43787.915063] usb 2-1.1: New USB device found, idVendor=0b95, idProduct=1790
[43787.915077] usb 2-1.1: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[43787.915085] usb 2-1.1: Product: AX88179
[43787.915093] usb 2-1.1: Manufacturer: ASIX Elec. Corp.
[43787.915100] usb 2-1.1: SerialNumber: 00000000000007
[43788.275539] ax88179_178a 2-1.1:1.0 eth0: register 'ax88179_178a' at usb-xhci-hcd.2.auto-1.1, ASIX AX88179 USB 3.0 Gigabit Ethernet, 00:23:55:0c:14:07
[43788.631056] IPv6: ADDRCONF(NETDEV_UP): eth0: link is not ready
[43790.548147] ax88179_178a 2-1.1:1.0 eth0: ax88179 - Link status is: 1
[43790.550074] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready

**Comment by tedm** [comment]
another way is to make up an sd card to boot from, and enable boot from usb. Here's a blog post on it, but I haven't tried it yet or written a book about it yet, but I have done the latter with Arch linux, should be similar, also, if you have a Murata wifi chipset, they may not do promiscuous mode, you may need to find a usb solution for that too.

https://citruspi.io/kali-on-chromebook/

**Comment by mark-hahn** [comment]
> you may need to find a usb solution 

I went ahead and ordered a usb wifi adapter for $15 that is known to work.  I burned out on chasing this down.  I'm not going to be using it very often and it will work in any computer.

For those curious, I'm not interested in stealing my neighbor's wifi.  (grin).  I'm sniffing for packets to my Harmony Remote hub to add IR capabilities to my homebrew entertainment system.
